app/views/pending_loans/show.html.erb

```diff
@@ -16,8 +16,8 @@
     <% else %>
       <% @pending_loans.each do |project, loan_amount| %>
         <tr id=<%= "project_#{project.id}" %>>
-          <td><%= project.tenant.organization %></td>
-          <td><%= link_to project.title, project_path(project) %></td>
+          <td><%= project.tenant.slug %></td>
```

Comment:
This one was fine before(project.tenant.organization)--for humans it's OK to have "Save the children"